### ISSUE TITLE
Update Dutch translations

### ISSUE THREAD

**Issue by evetion** [squidfunk/mkdocs-material#4955]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "nl",
  "action.edit": "Wijzig deze pagina",
  "action.skip": "Ga naar inhoud",
  "action.view": "Bron van deze pagina bekijken",
  "announce.dismiss": "Niet meer laten zien",
  "blog.archive": "Archief",
  "blog.categories": "Categorieën",
  "blog.categories.in": "in",
  "blog.continue": "Doorgaan met lezen",
  "blog.draft": "Concept",
  "blog.index": "Terug naar de inhoudsopgave",
  "blog.meta": "Metadata",
  "blog.references": "Gerelateerde links",
  "clipboard.copy": "Kopiëren naar klembord",
  "clipboard.copied": "Gekopieerd naar klembord",
  "consent.accept": "Accepteren",
  "consent.manage": "Instellingen",
  "consent.reject": "Afwijzen",
  "footer": "Footer",
  "footer.next": "Volgende",
  "footer.previous": "Vorige",
  "header": "Header",
  "meta.comments": "Reacties",
  "meta.source": "Bron",
  "nav": "Navigatie",
  "readtime.one": "1 min leestijd",
  "readtime.other": "# min leestijd",
  "rss.created": "RSS feed",
  "rss.updated": "RSS feed met geüpdatet inhoud",
  "search": "Zoeken",
  "search.placeholder": "Zoeken",
  "search.share": "Delen",
  "search.reset": "Leegmaken",
  "search.result.initializer": "Zoeken initialiseren",
  "search.result.placeholder": "Typ om te beginnen met zoeken",
  "search.result.none": "Geen overeenkomende documenten",
  "search.result.one": "1 overeenkomende document",
  "search.result.other": "# overeenkomende documenten",
  "search.result.more.one": "1 extra overeenkomst op deze pagina",
  "search.result.more.other": "# extra overeenkomsten op deze pagina",
  "search.result.term.missing": "Ontbreekt",
  "select.language": "Selecteer taal",
  "select.version": "Selecteer versie",
  "source": "Ga naar repository",
  "source.file.contributors": "Donateurs",
  "source.file.date.created": "Gecreëerd",
  "source.file.date.updated": "Laatst geüpdatet",
  "tabs": "Tabs",
  "toc": "Inhoudsopgave",
  "top": "Terug naar boven"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.

**Comment by evetion** [comment]
We could improve the translations even more when each key is given a context. A live demo with these fields highlighted would help. For example, the exact context for  `search.result.term.missing` would help with an accurate translation. It's `missing` in English, but the German versions indicates this only happens when something errors?

**Comment by squidfunk** [comment]
> We could improve the translations even more when each key is given a context.

We're working on that!

**Comment by squidfunk** [comment]
Added in 5f39cfb98.

**Comment by HonkingGoose** [comment]
This is wrong:

>   "source.file.contributors": "Donateurs",

The Dutch word "Donateurs" means persons who give money to some cause, generally charity. The closest English equivalent to "Donateurs" is sponsor or patron.

The Dutch dictionary "van Dale" says the correct translation is: "Bijdrager" for the singular form, and so "Bijdragers" for the Dutch plural.

https://www.vandale.nl/gratis-woordenboek/engels-nederlands/vertaling/contributors

```diff
-   "source.file.contributors": "Donateurs",
+   "source.file.contributors": "Bijdragers",
```

**Comment by HonkingGoose** [comment]
I'm also not 100% happy with "bijdragers". But it seems less wrong than "donateurs".

Would "vrijwilligers" work, as in people who volunteer?

How much physical space do we have in the elements that use the text? I imagine the translation should be of similar length as the English term?

**Comment by evetion** [comment]
> I'm also not 100% happy with "bijdragers". But it seems less wrong than "donateurs".
> 
> Would "vrijwilligers" work, as in people who volunteer?

But some are probably paid to do open source stuff. Let's go for bijdragers.

**Comment by HonkingGoose** [comment]
Let's go with `Bijdragers` then. I'll let @evetion create the pull request. 😉

**Comment by evetion** [comment]
See https://github.com/squidfunk/mkdocs-material/pull/4958

**Comment by squidfunk** [comment]
Released as part of [9.0.9](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.9)
